.github/workflows/CI.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - "1.40.0"
+          - "1.42.0"
```

Comment:
Why 1.42 instead of 1.41?

Reply:
I thought it was stabilized in 1.41 but it wasn't the case https://github.com/rust-lang/rust/pull/67659/